src/nodes/accessors/Normal.js

```diff
@@ -83,9 +83,13 @@ export const transformedNormalView = /*@__PURE__*/ ( Fn( ( builder ) => {
 
 	// Use getUV context to avoid side effects from nodes overwriting getUV in the context (e.g. EnvironmentNode)
 
-	return builder.context.setupNormal().context( { getUV: null } );
+	const node = builder.context.setupNormal().context( { getUV: null } );
 
-}, 'vec3' ).once() )().mul( faceDirection ).toVar( 'transformedNormalView' );
```

Comment:
@sunag What was the reasons to add `.mul( faceDirection )` after the `once()` call? The new code integrates the multiplication inside the `Fn` since the code needs access to the builder. Does that have any side effects?

Reply:
Um, looks like the current approach breaks the visuals a bunch of examples. Investigating...